### ISSUE TITLE
Starting with PHP

### ISSUE THREAD

**Issue by fernandofreamunde** [fernandofreamunde/challenge-github-blog#5]
PHP is a popular open-source server-side scripting language that is widely used for web development. If you're new to PHP, here's a simple guide to help you get started:

Install PHP: The first step to using PHP is to install it on your computer. You can download the installation file from the official PHP website. Follow the instructions provided to complete the installation process.

Learn the basics: Once you have installed PHP, it's time to learn the basics. Start by learning about PHP syntax, variables, data types, and control structures. You should also learn about PHP functions and how to use them.

Create a Hello World program: A Hello World program is a simple program that outputs the text "Hello, World!" to the browser. Create a new file and add the following code:

```php
<?php
  echo "Hello, World!";
?>
```

Save the file as "helloworld.php".

Run the program: To run the program, open a web browser and navigate to the file location on your local server. For example, if you saved the file in the "htdocs" folder of your XAMPP installation, you would navigate to "http://localhost/helloworld.php". The browser should display the text "Hello, World!".

Create a web page: PHP can also be used to create dynamic web pages. Create a new file and add the following code:

```php
<!DOCTYPE html>
<html>
<head>
  <title>My PHP Web Page</title>
</head>
<body>
  <h1>My PHP Web Page</h1>
  <?php
    echo "Today is " . date("l") . ".";
  ?>
</body>
</html>
```

Save the file as "index.php".

Run the web page: To run the web page, open a web browser and navigate to the file location on your local server. For example, if you saved the file in the "htdocs" folder of your XAMPP installation, you would navigate to "http://localhost/index.php". The browser should display the text "My PHP Web Page" and the current day of the week.
In conclusion, PHP is a powerful server-side scripting language that is widely used for web development. By following the steps outlined above, you can get started with PHP and start building your own dynamic web pages and applications.
